packages/flutter_tools/test/general.shard/android/android_project_migration_test.dart

```diff
@@ -44,6 +45,78 @@ zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 ''';
 
+String sampleModuleGradleBuildFile(String minSdkVersionString) {
+  return r'''
+plugins {
+    id "com.android.application"
+    id "kotlin-android"
+    id "dev.flutter.flutter-gradle-plugin"
+}
+
+def localProperties = new Properties()
+def localPropertiesFile = rootProject.file('local.properties')
+if (localPropertiesFile.exists()) {
+    localPropertiesFile.withReader('UTF-8') { reader ->
+        localProperties.load(reader)
+    }
+}
+
+def flutterVersionCode = localProperties.getProperty('flutter.versionCode')
+if (flutterVersionCode == null) {
+    flutterVersionCode = '1'
+}
+
+def flutterVersionName = localProperties.getProperty('flutter.versionName')
+if (flutterVersionName == null) {
+    flutterVersionName = '1.0'
+}
+
+android {
+    namespace "com.example.asset_sample"
+    compileSdkVersion flutter.compileSdkVersion
+    ndkVersion flutter.ndkVersion
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+    }
+
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        applicationId "com.example.asset_sample"
+        // You can update the following values to match your application needs.
+        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
+        ''' + minSdkVersionString + r'''
+        targetSdkVersion flutter.targetSdkVersion
```

Comment:
```suggestion
        ''' + minSdkVersionString + r'''

        targetSdkVersion flutter.targetSdkVersion
```

The `'''` syntax eats the first newline, which is often convenient but means that this code doesn't produce the string it looks like it does. As is, the string looks like
```
…
        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
        minSdkVersion 16        targetSdkVersion flutter.targetSdkVersion
        versionCode flutterVersionCode.toInteger()
…
```
with the `targetSdkVersion` line squished onto the `minSdkVersion` line.

Reply:
Thanks for catching this! I've added the appropriate line break.